doc/cookbook/source/references.bib

```diff
@@ -24,3 +24,23 @@ @book{Rasmussen2005GPM
   year = {2005},
   publisher = {The MIT Press}
 }
+@article{chang2011libsvm,
+  title={LIBSVM: a library for support vector machines},
+  author={Chang, Chih-Chung and Lin, Chih-Jen},
+  journal={ACM Transactions on Intelligent Systems and Technology (TIST)},
+  volume={2},
+  number={3},
+  pages={27},
+  year={2011},
+  publisher={ACM}
```

Comment:
Another good reference here is the book "Learning with kernels" By Schölkopf and Smola